modules/schema/src/test/java/org/apache/ignite/internal/schema/marshaller/KvMarshallerTest.java

```diff
@@ -161,7 +161,14 @@ public void pojoWithFieldsOfAllTypes(MarshallerFactory factory) throws Marshalle
     @ParameterizedTest
     @MethodSource("marshallerFactoryProvider")
     public void narrowType(MarshallerFactory factory) throws MarshallerException {
-        Column[] cols = columnsAllTypes();
+        Column[] cols = new Column[]{
+                new Column("primitiveIntCol", INT32, false),
+                new Column("primitiveLongCol", INT64, false),
+                new Column("primitiveFloatCol", FLOAT, false),
+                new Column("primitiveDoubleCol", DOUBLE, false),
+                new Column("stringCol", STRING, false),
+                new Column("uuidCol", UUID, false),
+        };
```

Comment:
Initially, test check we could map narrow type where amount of fields > amount of columns.
Now, amount of fields equal amount of columns.

Reply:
All types used for value.